src/Microsoft.Health.Fhir.Web/appsettings.json

```diff
@@ -25,7 +25,11 @@
         "Host": null,
         "Key": null,
         "DatabaseId": "health",
-        "CollectionId": "fhir",
+        "InitialDatabaseThroughput": null,  
```

Comment:
The PR release didn't work when this was set to 400. I will need to look into why this failed going to an actual Cosmos DB account, but worked when going to the local emulator.